teams/lang-docs.toml

```diff
@@ -0,0 +1,22 @@
+name = "lang-docs"
+subteam-of = "lang"
+
+[people]
+leads = ["ehuss", "JohnTitor"]
+members = [
+    "ehuss",
+    "Havvy",
+    "JohnTitor",
+    "matthewjasper",
+]
+alumni = [
+    "Centril",
+]
+
+[[github]]
+orgs = ["rust-lang"]
+
+[website]
+name = "lang-docs team"
+description = "Developing and writing the docs related to the lang team"
+zulip-stream = "t-lang/docs"
```

Comment:
This link is broken.  The zulip stream is `t-lang/doc`.

Reply:
Oops, thanks. Will fix it soon.